cypress/e2e/digital-certificates.cy.js

```diff
@@ -39,35 +40,23 @@ describe('Digital Certificates spec', () => {
 
     // Act
     cy.get(selectors.form.digitalCertificateName).clear()
-    cy.get(selectors.form.digitalCertificateName).type('EntityName')
+    cy.get(selectors.form.digitalCertificateName).type(digitalCertificateName)
     cy.get(selectors.form.submitButton).click()
 
     // Assert
-    cy.get(selectors.toast.createSuccessMessage).should(
-      'have.text',
-      'Your digital certificate has been created!'
-    )
-    cy.get(selectors.form.editPageTitle).should(
-      'have.text',
-      'Edit Digital Certificate'
-    )
+    cy.verifyToast('success', 'Your digital certificate has been created!')
+    cy.get(selectors.form.editPageTitle).should('have.text', 'Edit Digital Certificate')
     cy.get(selectors.list.breadcumbReturnToList).click()
     cy.get(selectors.list.searchInput).clear()
-    cy.get(selectors.list.searchInput).type('EntityName')
-    cy.get(selectors.list.filteredRow.nameColumn).should('have.text', 'EntityName')
-    cy.get(selectors.list.filteredRow.statusColum).should(
-      'have.text',
-      'Pending'
-    )
+    cy.get(selectors.list.searchInput).type(digitalCertificateName)
+    cy.get(selectors.list.filteredRow.nameColumn).should('have.text', digitalCertificateName)
+    cy.get(selectors.list.filteredRow.statusColum).should('have.text', 'Pending')
```

Comment:
Minor typo. Could you please double check if this won't cause troubles?
```suggestion
    cy.get(selectors.list.filteredRow.statusColumn).should('have.text', 'Pending')
```